rust/lon/src/cli.rs

```diff
@@ -52,7 +52,10 @@ enum Commands {
     /// When you change the revision,  the source is locked to this revision.
     Modify(ModifyArgs),
     /// Remove an existing source
-    Remove(RemoveArgs),
+    Remove(SourceArgs),
+    /// Freeze an existing source
+    Freeze(SourceArgs),
+    Thaw(SourceArgs),
```

Comment:
```suggestion
    Unfreeze(SourceArgs),
```

Let's call this `unfreeze` instead of `thaw`:

- unfreeze is easier to understand (esp. for non-native speakers)
- thaw describes a process (think of putting something out of your freezer to slowly let it thaw) while unfreezing is instantaneous (this is obviously just my personal opinion)